packages/grafana-ui/src/components/Slider/RangeSlider.story.tsx

```diff
@@ -27,7 +27,15 @@ const meta: ComponentMeta<typeof RangeSlider> = {
 export const Basic: ComponentStory<typeof RangeSlider> = (args) => {
   return (
     <div style={{ width: '200px', height: '200px' }}>
-      <RangeSlider value={[10, 20]} {...args} />
+      <RangeSlider value={[10, 62]} {...args} />
+    </div>
+  );
+};
+
+export const Vertical: ComponentStory<typeof RangeSlider> = (args) => {
```

Comment:
is this supposed to be a vertical slider? story still shows horizontal for me:
![image](https://user-images.githubusercontent.com/20999846/196472133-b8f0c823-380a-4f38-acac-44835e6a2b95.png)

Reply:
should be fixed now.